polynote-kernel/src/main/scala/polynote/config/PolynoteConfig.scala

```diff
@@ -163,10 +162,21 @@ object Credentials {
   implicit val decoder: Decoder[Credentials] = deriveDecoder
 }
 
+final case class ScalaVersionConfig(
+  versionNumber: String,
+  versionProperties: ShortMap[String, String] = ShortMap(Map.empty[String, String])
```

Comment:
Could we add a top-level `sparkSubmitArgs` key, like `SparkPropertySet`? (I know the notebook config sticks it in the spark properties but I think that was a bit of a hack).

Reply:
[Added](https://github.com/polynote/polynote/pull/1432/files#diff-95c5c982fc4bf39d8e0d97de3fdb1ed4601fd8c78723869a676fb38a2f41f728R165-R169)!